exodus_analyze.py

```diff
@@ -15,32 +15,40 @@ def create_json_report(self):
                 'uaid': self.get_application_universal_id(),
                 'name': self.get_app_name(),
                 'permissions': self.get_permissions(),
-                'libraries': [l.decode('utf-8') for l in self.get_libraries()],
+                'libraries': [l for l in self.get_libraries()],
```

Comment:
if using python3, either the old or the new code should not work

Reply:
I tested it with python3.6 and it works well with the new code.

Reply:
my mistake it was a bug corrected